webapp/src/components/rhs/rhs_checklist_list.tsx

```diff
@@ -98,6 +98,14 @@ const RHSChecklistList = (props: Props) => {
     const filterOptions = makeFilterOptions(checklistItemsFilter, preferredName);
     const overdueTasksNum = overdueTasks(props.playbookRun.checklists);
 
+    // Hide overdueOnly filter if there are no overdue tasks anymore
```

Comment:
Another suggestion about comment text:
```suggestion
    // Cancel overdueOnly filter if there are no overdue tasks anymore
```